src/api/getTimezoneOptions.ts

```diff
@@ -0,0 +1,11 @@
+import {API_ENDPOINT} from '@env';
+import axios from 'axios';
+
+export default async function getTimezoneOptions(
+  userTz: string,
+): GetTimezoneOptionsResult {
+  const {data} = await axios.get(
+    `${API_ENDPOINT}/get-tz-options?userTz=${userTz}`,
```

Comment:
suggestion: we can have a `url.constants.ts` file where we can have all the API endpoints. can be implemented later

Reply:
Should we keep the API endpoints in env or a .ts file?

Reply:
the main API host can be in `.env`. Remaining endpoints which we are writing in these files (example: `/get-tz-options?userTz=`) can be in a constants file.